core/src/main/java/org/infinispan/xsite/conflict/XSiteConflictResolver.java

```diff
@@ -0,0 +1,15 @@
+package org.infinispan.xsite.conflict;
```

Comment:
We should probably create a `org.infinispan.xsite.spi` package for the SPI. This is how we separate the SPI from the implementation in `org.infinispan.persistence`.